benchmarks/1-vu.script.js

```diff
@@ -0,0 +1,25 @@
+import http from "k6/http";
+import { check, sleep } from "k6";
+import { describe } from 'https://jslib.k6.io/expect/0.0.4/index.js';
+
+// Test configuration
+export const options = {
+  thresholds: {
+    // http request will be crossed with 
+    http_req_duration: ["p(10) > 30000"],
+  },
+  // better to make it longer to cross with validators update
+  // we will need this benchmark to check how long 1 request is handled depends on kapi job execution
+  duration: "80s",
+  vus: 1
+};
+
+// Simulated user behavior
+export default function () {
+  // this endpoint process the 
```

Comment:
Maybe there is a typo? Like `process the ...`

Reply:
deleted